.github/workflows/rosa-cost-report.yml

```diff
@@ -0,0 +1,30 @@
+name: ROSA/AWS - Cost Reporting
+
+on:
+  schedule:
+    - cron: '0 12 * * Sun' # Runs At 12:00 PM UTC on Every Sunday
+  workflow_dispatch:
+
+jobs:
+  report:
+    name: Report on ROSA and AWS Costs for the current YTD, MTD and WTD
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
```

Comment:
I assume that if we don't pass on the ROSA_TOKEN, the call will fail. Assuming that the AWS CLI is already installed on the action, could you configure it straight away?

Reply:
yup, its a hard requirement for the rosa-cli-setup.sh script so will add that back.